be/src/util/buffered_stream.cpp

```diff
@@ -2,14 +2,15 @@
 
 #include "util/buffered_stream.h"
 
+#include "common/config.h"
 #include "env/env.h"
 #include "util/bit_util.h"
 
 namespace starrocks {
 
 BufferedInputStream::BufferedInputStream(RandomAccessFile* file, uint64_t offset, uint64_t length)
         : _file(file), _offset(offset), _end_offset(offset + length) {
-    _reserve(8 * 1024 * 1024);
+    _reserve(config::buffer_stream_reserve_size);
```

Comment:
This can not be changed too small.
When it is used by HDFS scanner. Too small buffer means too many RPC.
So we should keep buffer size large enough.

Reply:
@imay 
If a column will retain an 8M buffer, each scanner in a 16 columns table should retain at least 128M.
we most likely be scheduling 50 scanners at the same time. so we should reserve 6.4G.
The 8K buffer may be a bit small. I try to choose a suitable value.